### DISCUSSION TITLE
PC端 磁力解析报错

### DISCUSSION THREAD

**Discussion by rurutia74** [xiaohucode/yidaRule#162] [Q&A] [answered]
安卓和IOS端正常，PC端提示错误
![图像_2024-05-14_163237722](https://github.com/xiaohucode/yidaRule/assets/169764964/97a838e2-eb6d-4b2c-a6bd-213521767692)

**Comment by xiaohucode** [comment, marked as answer]
解压使用，检查目录是否存在 【bt_server-x64.dll】这个文件

**Reply by rurutia74** [reply]
解压了的，也有这个文件，但是还是会报错

**Reply by xiaohucode** [reply]
> 解压了的，也有这个文件，但是还是会报错

缺少gcc运行库,这个问题将在下个版本中修复

已修复！pc平台重新下载[yidaApp-1.1.26-windows-x86_64-fix.zip](https://github.com/xiaohucode/yidaRule/releases/download/1.1.26/yidaApp-1.1.26-windows-x86_64-fix.zip)

**Reply by rurutia74** [reply]
好的谢谢大佬 ，在线等更新

**Comment by fuxin-mail** [comment]
你们的磁力能看吗？我这个一直转圈呢
